templates/neutronapi/httpd/httpd.conf

```diff
@@ -0,0 +1,23 @@
+ServerTokens Prod
+ServerSignature Off
+TraceEnable Off
+ServerRoot "/etc/httpd"
+ServerName "neutron.openstack.svc"
```

Comment:
seems openstack here is namespace name, and neutron CR name, we shouldn't hard code these